push-messaging-and-notifications/main.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+var API_KEY = 'AIzaSyBBh4ddPa96rQQNxqiq_qQj7sq1JdsNQUQ';
+
+var isPushEnabled = false;
+
+function sendToServer(subscriptionId, endpoint) {
+  // The curl command to trigger a push message straight from GCM
+  var curlCommand = 'curl --header "Authorization: key=' + API_KEY +
+    '" --header Content-Type:"application/json" ' + endpoint + 
+    ' -d "{\\"registration_ids\\":[\\"' + subscriptionId + '\\"]}"';
+
+  // Below is a curl command to test sending a push message
+  console.log(curlCommand);
```

Comment:
I can do if you want but if the user disables notifications  / unsubscribes I then have to manage the state of the message which is what I wanted to avoid.

What would you like?